include/nuttx/spinlock.h

```diff
@@ -300,7 +300,7 @@ void spin_unlock_wo_note(FAR volatile spinlock_t *lock);
  *
  ****************************************************************************/
 
-/* bool spin_islocked(FAR spinlock_t lock); */
+/* bool spin_islocked(FAR spinlock_t *lock); */
```

Comment:
Please add a new line after the comment line

Reply:
We have many places similar to this. I just fixed missing `*`. Let's add newlines as a separate change